testing/MLDB-1428-text-sparse-output.py

```diff
@@ -0,0 +1,103 @@
+#
+# MLDB-1428-text-sparse-output.py
+# Mathieu Bolduc, 2016-02-29
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+
+import unittest
+class HavingTest(unittest.TestCase):
+
+    def test_sparse(self):
+        res = mldb.put('/v1/procedures/import_reddit', { 
+            "type": "import.text",  
+            "params": { 
+                "dataFileUrl": "file://mldb/testing/dataset/iris.data",
+                'encoding' : 'latin1',
+                "select": "*",
+                'outputDataset': {'id': 'iris', 'type': 'sparse.mutable'},
+                 'headers': ["a", "b", "c", "d", "label"],
+                'runOnCreation': True
+            } 
+        })
+
+        res = mldb.query('SELECT * FROM iris limit 1')
+
+        expected = [["_rowName",
```

Comment:
Since new lines are not considered in comparison, I usually write it this way
```
["_rowName", a",  "b",  "c", "d", "label"],
 [ "97",  5.7,  2.9,  4.2, 1.3, "Iris-versicolor"]
```
which I find easier to read and reduce the length of the tests.